src/PositionManager.sol

```diff
@@ -270,10 +276,13 @@ contract PositionManager is
         uint256 liquidity = uint256(getPositionLiquidity(tokenId, config));
 
         BalanceDelta liquidityDelta;
+        BalanceDelta feesAccrued;
```

Comment:
why are these defined outside of the if statement?

Reply:
moved!